CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Add a series of components to power breadcrumbs.
 - Add a frontend component to use SVG icons in Marsha.
+- Add a ngrok makefile task to launch it
+- Inject current ngrok url to terraform configuration through env vars
```

Comment:
I didn't really understand what you are doing when reading these 2 sentences. I propose something like:
```suggestion
- Add ngrok to serve marsha on a public domain and automate CloudFront configuration via terraform when needed
```

Reply:
You're right, final usage wasn't clear.